Content/posts/introducing-docc-to-the-swift-package-index.md

```diff
@@ -0,0 +1,86 @@
+- `TODO despite some initial updates still a bit tricky to set up for hosting`
+
+[![A screenshot showing a sample documentation page.](/images/documentation-tca.png)](https://swiftpackageindex.com/pointfreeco/swift-composable-architecture/main/documentation/composablearchitecture)
+
+## How does the Swift Package Index help?
+We’ve updated our build system to not only build packages but also generate their documentation and make it available from the package page itself. If a package comes with documentation, you can find a link to it in the sidebar.
+
+![A screenshot showing the documentation link on a package page.](/images/documentation-link.png)
+
+There are only two steps you, as the package author, need to take to host your documentation on the Swift Package Index:
+
+1. Add the [swift-docc-plugin][1] documentation plugin to your `Package.swift` manifest file, if you haven’t done so already.
+2. Create a `.spi.yml` file in the root of your package repository telling us which targets to build and host documentation for.
+
+### Update your package manifest
+
+The first step you have likely already done if you’ve worked with DocC locally. If not, here is the snippet you can simply add to the end of your `Package.swift` manifest:
+
+```swift
+#if swift(>=5.6)
+  // Add the documentation compiler plugin if possible
+  package.dependencies.append(
+    .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0")
+  )
+#endif
+```
+
+You can add this even if the tools-version of your manifest is for an earlier version of Swift than 5.6. The `#if swift(>=5.6)` will make sure it doesn’t interfere with your manifest when run with older Swift versions.
+
+### Add an SPI manifest
+
+The second step is equally simple: Create the following `.spi.yml` file in the root of your repository:
+
+```yaml
+version: 1
+builder:
+  configs:
+  - documentation_targets: [Target]
+```
+
+This will tell us to generate documentation for the target `Target`. This can (and typically is) your package’s main or only target but it can also be a dedicated target that contains documentation markdown files.
+
+Substitute your target here – or even list several if your documentation is set up to have multiple.
+
+## Documentation platform
+
+By default we are generating documentation on macOS. If your package requires the documentation generation to be run for a certain platform, for instance iOS, you can control this by specifying the platform as well:
+
+```yaml
+version: 1
+builder:
+  configs:
+  - documentation_targets: [DocumentationTarget]
+    platform: ios
+```
+
+## Update frequency
+
+We are building documentation for a package’s default branch as well as any releases. Please note that we collate builds on the default branch over 24 hours in order to reduce the number of builds we have to process for fast-changing repositories.
+
+That means that it will take 24 hours for initial documentation to appear unless you tag a new release.
```

Comment:
I just added "unless you tag a new release" but now I've realised that while do build the docs in that case that won't make the documentation link work unless we link to release docs.